README.md

```diff
@@ -30,8 +30,11 @@ into your your app's `build.gradle` dependencies. `mediarouter`version must matc
 // Require the module
 import Chromecast from 'react-native-google-cast';
 
-// Init Chromecast SDK and starts looking for devices
-Chromecast.startScan();
+// Init Chromecast SDK and starts looking for devices (uses DEFAULT APP ID)
+Chromecast.startDefaultScan();
+
+// Init Chromecast SDK and starts looking for devices using registered APP ID
+Chromecast.startScan(APP_ID);
```

Comment:
So, there are two ways here

1) In Objective C there are no optional parameters. But, you can have multiple functions with the same name, that receives different arguments.

It would be something like 

```RCT_EXPORT_METHOD(startScan)
      RCT_EXPORT_METHOD(startScan:(NSString *)appID) 
```

2) We can rely on JS! By using default parameters (in index.js)
```
startScan: function (appId: string = '') {
}

Personally, I prefer the first option :) You need to change the name from startDefaultScan to startScan in GoogleCast.m

Just test if it is working as expected. I'm not sure if idea of multiple functions with same name but different parameters works with @ReactMethod